addons/mail/static/src/models/chat_window/chat_window.js

```diff
@@ -138,6 +139,21 @@ function factory(dependencies) {
             this.update({ isMemberListOpened: true });
         }
 
+        /**
+         * Handles click on the "invite" button.
+         *
+         * @param {MouseEvent} ev
+         */
+        onClickInviteButton(ev) {
+            if (this.threadView.channelInvitationForm.component) {
```

Comment:
I would not guard at all, or at most `threadView.channelInvitationForm`.
I wouldn't take `.component`.
It's strange to allow click on invite when the component is no longer there, but I feel it doesn't matter as long as the intent is to click on invite button and the rest of code don't need a component.

Reply:
I reused the existing code, this guard was part of the function.